crates/sui-core/src/state_accumulator.rs

```diff
@@ -363,27 +426,32 @@ impl StateAccumulator {
         Ok(root_state_hash)
     }
 
-    /// Returns the result of accumulatng the live object set, without side effects
-    pub fn accumulate_live_object_set(&self) -> Accumulator {
+    /// Returns the result of accumulating the live object set, without side effects
+    pub fn accumulate_live_object_set(&self, include_wrapped_tombstone: bool) -> Accumulator {
```

Comment:
it seems like it would make sense to make this a parameter to iter_live_object set